PostgreSQL/index.js

```diff
@@ -1,5 +1,8 @@
 const { Client } = require('pg');
-const info = require('./info.js'); // should refactor to use a .env file later
-const client = new Client(info.connection);
+const client = new Client({
+  host: /* HOST */,
+  user: /* USER */,
+  password: /* PW */,
```

Comment:
.env variables are quick to learn. LIke 30minutes should be enough time for you to block out and read about the couple of methods used to implement them.